libbpf-rs/tests/test_streams.rs

```diff
@@ -0,0 +1,66 @@
+//! Tests for BPF program streams (stdout/stderr).
+
+mod common;
+
+use std::io::Read;
+
+use libbpf_rs::ProgramInput;
+use test_tag::tag;
+
+use crate::common::get_prog_mut;
+use crate::common::get_test_object;
+
+/// Test that we can read from a BPF program's stdout stream.
+///
+/// This test loads a BPF program that triggers the streams mechanism,
+/// runs it, and then attempts to read from the stdout stream.
+#[tag(root)]
+#[test]
+fn test_stream_stdout_read() {
+    let mut obj = get_test_object("stream.bpf.o");
+    let prog = get_prog_mut(&mut obj, "trigger_streams");
+
+    let input = ProgramInput::default();
+    let _output = prog.test_run(input).unwrap();
+
+    let mut stdout = prog.stdout();
+    let mut buf = [0u8; 1024];
+
+    // The read itself should succeed and return 0 bytes
```

Comment:
Am I misreading or does the comment state the opposite of what is being asserted?